handlers/httpcontrollers/controller.go

```diff
@@ -0,0 +1,73 @@
+package httpcontrollers
+
+import (
+	"log"
+	"net/http"
+	"riot-developer-proxy/internal/domain/entities"
+	"riot-developer-proxy/internal/domain/services"
+
+	"github.com/labstack/echo/v4"
+)
+
+type HTTPController struct {
+	SummonerService *services.SummonerService
+}
+
+const INTERNAL_SERVER_ERROR_MESSAGE = "Internal Server Error"
+
+type SummonerResponse struct {
+	ID            string                 `json:"id"`
+	Name          string                 `json:"name"`
+	ProfileIconID int                    `json:"profileIconId"`
+	Level         int                    `json:"summonerLevel"`
+	RevisionDate  string                 `json:"revisionDate"`
+	LeagueEntries *[]LeagueEntryResponse `json:"leagueEntries"`
```

Comment:
Slices are already reference types, don't need to make a pointer to one

Reply:
Done